shared/tracker/bio.render.desktop.js

```diff
@@ -16,45 +16,51 @@ import type {BioProps} from './bio.render'
 export default class BioRender extends Component {
   props: BioProps;
 
-  onClickAvatar () {
+  _onClickAvatar () {
```

Comment:
turns out we have munge_underscores set to true already in our flowconfig so flow will enforce this